toolbox/py_xnvme_pkg/setup.py

```diff
@@ -0,0 +1,43 @@
+import codecs
+import glob
+import os
+
+import setuptools
+
+
+def read(*parts):
+    """Read parts to use a e.g. long_description"""
+
+    here = os.path.abspath(os.path.dirname(__file__))
+
+    # intentionally *not* adding an encoding option to open, See:
+    #   https://github.com/pypa/virtualenv/issues/201#issuecomment-3145690
+    with codecs.open(os.path.join(here, *parts), "r") as pfp:
+        return pfp.read()
+
+
+setuptools.setup(
+    name="py-xNVMe",
```

Comment:
This is the name for `pip uninstall [name]`

Reply:
Right, fixing that.